src/ape_ethereum/ecosystem.py

```diff
@@ -97,6 +97,7 @@ class NetworkConfig(PluginConfig):
     block_time: int = 0
     transaction_acceptance_timeout: int = DEFAULT_TRANSACTION_ACCEPTANCE_TIMEOUT
     default_transaction_type: TransactionType = TransactionType.DYNAMIC
+    max_receipt_retries = 20
```

Comment:
(see my other comment for more context)

```suggestion
    max_receipt_retries: int = MAX_RETRIES_TX
```